Now we have a PBD file in the installer.
However I am unsure if the source files and line numbers are part of it.
At least @abseits has not provided a backlog without this info.

Compiler Flags:

/TP /W3 /GR /EHsc /MD /Zi /O2 /Ob1 /DNDEBUG /fp:fast /Gy /O2 /GL /W3 /c -std:c++17 /showIncludes /FoCMakeFiles\mixxx-lib.dir

Linker Flags are:

/implib:mixxx.lib /pdb:mixxx.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:windows  /OPT:REF /OPT:ICF /LTCG:NOSTATUS /entry:mainCRTStartup 


LINK : warning LNK4075: ignoring '/INCREMENTAL' due to '/LTCG' specification

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1348,7 +1348,7 @@ if(UNIX AND NOT APPLE)
 endif()
 
 if(MSVC)
-  # install debug symbols if any where generated
+  # install debug symbols if any were generated
   install(
     FILES $<TARGET_PDB_FILE:mixxx>
     CONFIGURATIONS Debug RelWithDebInfo

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1353,7 +1353,7 @@ if(MSVC)
     FILES $<TARGET_PDB_FILE:mixxx>
     CONFIGURATIONS Debug RelWithDebInfo
     DESTINATION "${MIXXX_INSTALL_BINDIR}"
-    COMPONENT PDB
+    COMPONENT "Debug Information (PDB)"
   )
 endif()
 

